Source/ALS/Public/Utility/AlsMath.h

```diff
@@ -99,8 +99,7 @@ class ALS_API UAlsMath : public UBlueprintFunctionLibrary
 
 	// Remaps the angle from the [175, 180] range to [-185, -180]. Used to
 	// make the character rotate counterclockwise during a 180 degree turn.
-	template <typename ValueType> requires std::is_floating_point_v<ValueType>
-	static constexpr float RemapAngleForCounterClockwiseRotation(ValueType Angle);
+	static constexpr float RemapAngleForCounterClockwiseRotation(float Angle);
```

Comment:
Revert the template back and replace the return value type from `float` to `ValueType` instead.

Reply:
This change is for the warning in the implementation